features/appSettings/components/Entries/README.md

```
@@ -0,0 +1,246 @@
+- Layout structure (label, control, optional icon)
+- Theme integration
+- Accessibility support
+
+```typescript
+interface BaseEntryProps {
+ label: string;
+ description?: string;
+ value: any;
+ onChange: (value: any) => void;
+ icon?: ReactNode;
+ disabled?: boolean;
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)